From f5a72ada94ad444605514c7931e00abbe5f9fa39 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Fri, 6 Sep 2013 15:23:11 +0800
Subject: [PATCH 422/505] ENGR00278514 mxc als test: Remove the exclude out

The mxc_als_test is excluded in the Makefile. It causes
the mxc_als_test is not built into unit_test. Remove the
exclude list out.

Signed-off-by: Luwei Zhou <b45643@freescale.com>
---
 test/mxc_als_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_als_test/Makefile b/test/mxc_als_test/Makefile
index ece8207..56aaf8a 100644
--- a/test/mxc_als_test/Makefile
+++ b/test/mxc_als_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX6Q
+EXCLUDE_LIST:=
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_isl29023.out
-- 
1.8.0

